src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -105,6 +117,8 @@ protected virtual Task<TResponse> HandleDelegatedResponseAsync(TResponse delegat
             return default;
         }
 
+        var correlationId = Guid.NewGuid();
+        using var __ = _telemetryReporter?.TrackLspRequest(LspTarget, LanguageServerConstants.RazorLanguageServerName, correlationId);
```

Comment:
I thought we wanted this to be opt-in, or we'll be reporting too much telemetry?

Also, should the call be moved to under the guard clauses? Seems like we'd be reporting telemetry for requests we didn't do anything with, which would skew  our data.